src/constants/sfx.asm

```diff
@@ -2,278 +2,278 @@
 
 ; Values for wMusicTrackToPlay
 
-MUSIC_NONE                                      equ $00 ; This value means: Don't play a different track.
```

Comment:
This comment seems useful to me; it clarifies that it doesn't mean that _no music_ will be played (ie. music will stop).